spec/unit/matrix-client.spec.ts

```diff
@@ -1297,4 +1297,37 @@ describe("MatrixClient", function() {
             expect(result!.aliases).toEqual(response.aliases);
         });
     });
+
+    describe("ignoring invites", () => {
+        it('stores ignored invites', async function() {
```

Comment:
we should be using `should`, and arrow functions:
```suggestion
        it('should store ignored invites', async () => {
```